docs/adr/0012-biometrics-module-system.md

```
@@ -0,0 +1,230 @@
+  description: string
+  provides: string[]    // e.g. ["vitals.heartRate", "vitals.hrv"]
+  writes: string[]      // DB table names: ["vitals"]
+  service: string       // systemd unit name
+  language: string      // informational: "python" | "rust" | etc.
+  minVersion?: string   // minimum sleepypod-core version required
+}
+```
+
+### Bundled modules (`modules/`)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)